packages/sdk-react/src/components/userRoles/UserRolesTable/PermissionsCell/PermissionsCell.tsx

```diff
@@ -21,7 +20,7 @@ export const PermissionsCell = ({ permissions }: PermissionsCellProps) => {
 
   return (
     <Grid container>
-      {permissions.objects?.map((object, index) => {
+      {permissions.objects?.map((object) => {
         if (object.object_type) {
```

Comment:
You can filter results to avoid conditions in the map.

Reply:
fixed

Reply:
I don't see this change. I mean something like this:

```typescript
<Grid container>
  {permissions.objects
    .filter((object) => !!object.object_type)
    .map(({ object_type, actions }) => (
      <Grid item container key={object_type}>
        <Grid item xs={6}>
          {getPermissionToLabelMap(i18n)[object_type!]}
        </Grid>
        <Grid item xs={6}>
          {actions && <Permission actions={actions} />}
        </Grid>
      </Grid>
    ))}
</Grid>
```

Reply:
it doesn't show the diff for some reason. 
please see https://github.com/team-monite/monite-sdk/pull/25/files#diff-c2b37801c8fced54496c7d47e1e94bf58ede864214cb4086b64b565aac95a05aR27
The `permissions` object contains only one optional key, `objects`. In my opinion, `.filter` is not necessary here. I added a more explicit check.

Reply:
updated, fixed